docs/guide/example-setups/typescript.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to use Sublime for TypeScript development
+---
+
+Sublime Text 4 [ships with built-in TypeScript syntax](https://www.sublimetext.com/blog/articles/sublime-text-4).
+
+If you’re looking for highlighting plus core Sublime [completion](../extensibility/completions.html) and [“Go to Definition”](https://www.sublimetext.com/docs/indexing.html) behaviors, you may not need anything more that what’s built in!  (The [`TypeScript Syntax`](https://packagecontrol.io/packages/TypeScript%20Syntax) package can be used for ST3.)
+
+There are several options for more advanced workflows, if you’re looking for more sophisticated autocomplete or error-detection features.  Many function similarly, so you might want to give each a try to see which fits your preferences best:
```

Comment:
I'm of the opinion that there is really only one workflow and that is LSP + LSP-typescript.

Reply:
Good feedback! Revised to make this clearer.